CHANGELOG.md

```diff
@@ -11,6 +11,7 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 ### Added
 
 - Added `ErrorHandlerFunc` to use a function as an `"go.opentelemetry.io/otel".ErrorHandler`. (#2149)
+- Update which go modules have a version() func with a version.go file. (#2143)
```

Comment:
Can we remove this? These are not exported changes and we probably do not need to communicate them to the end user.